package.json

```diff
@@ -11,8 +11,8 @@
   "scripts": {
     "build": "npm run build:js && npm run build:types",
     "build:js": "ipjs build --tests --main && npm run build:copy",
-    "build:copy": "mkdir -p dist/examples/ && cp -a tsconfig.json *.js *.ts lib/ test/ dist/ && cp examples/*.* dist/examples/",
-    "build:types": "npm run build:copy && cd dist && tsc --build",
+    "build:copy": "mkdir -p dist/examples/ && cp -a tsconfig.json *.js *.ts lib test examples dist/",
```

Comment:
the `examples/*.*` is a bit weird, but it's trying to avoid a node_modules folder in there that's created as part of the test run, so it needs to come back

Reply:
Restored.